src/provider/sequencer.ts

```diff
@@ -74,9 +75,13 @@ export class SequencerProvider implements ProviderInterface {
       this.gatewayUrl = urljoin(this.baseUrl, 'gateway');
     } else {
       this.baseUrl = optionsOrProvider.baseUrl;
-      this.feederGatewayUrl =
-        optionsOrProvider.feederGatewayUrl ?? urljoin(this.baseUrl, 'feeder_gateway');
-      this.gatewayUrl = optionsOrProvider.gatewayUrl ?? urljoin(this.baseUrl, 'gateway');
+      this.feederGatewayUrl = this.parseUrlOrPath(
+        this.baseUrl,
+        'feeder_gateway',
+        optionsOrProvider.feederGatewayUrl
+      );
+      this.gatewayUrl = this.parseUrlOrPath(this.baseUrl, 'gateway', optionsOrProvider.gatewayUrl);
+
```

Comment:
It would be wise to extract endpoints strings to the constants/constants file

Reply:
I think it is out of the scope of this PR